arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,870 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the possible operations on the Microsoft.VisualStudio resource provider.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getOperationResources_List": {
+                "$ref": "../examples/GetOperations.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.visualstudio/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "nameAvailability"
+        ],
+        "summary": "Accounts_CheckNameAvailability",
+        "description": "Checks if the specified Visual Studio Team Services account name is available. Resource name can be either an account name or an account name and PUID.",
+        "operationId": "accounts_CheckNameAvailability",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The request data.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NameAvailabilityPostRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the result of the name availability check.",
+            "schema": {
+              "$ref": "#/definitions/NameAvailabilityResponse"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "checkNameAvailability": {
+                "$ref": "../examples/CheckNameAvailability.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account": {
+      "get": {
+        "tags": [
+          "accountResourceGroup"
+        ],
+        "summary": "Accounts_List",
+        "description": "Gets the Visual Studio Team Services account resources linked to the specified Azure subscription.",
+        "operationId": "accounts_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the Visual Studio Team Services account resources linked to the Azure subscription.",
+            "schema": {
+              "$ref": "#/definitions/AccountResourceGroupGetResponse"
```

Comment:
To be consistent with other swagger docs, a better name for this type would be `AccountListResult`.